pennylane_lightning/lightning_qubit.py

```diff
@@ -744,6 +744,11 @@ def probability(self, wires=None, shot_range=None, bin_size=None):
         Returns:
             array[float]: list of the probabilities
         """
+        if wires and len(wires) > 1 and (not np.all(list(wires)[:-1] <= list(wires)[1:])):
```

Comment:
same comment as in the [kokkos PR](https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/41#discussion_r1113490294) - this check should be done on `device_wires` below in case the device itself has a weird wire order

Reply:
Yep, done. Thanks @timmysilv

Reply:
Done.